merlin/models/tf/models/utils.py

```diff
@@ -1,19 +0,0 @@
-from typing import List, Optional, Union
```

Comment:
I think we need to adjust the RST stubs in docs/source/api to reflect that this file is deleted (its the reason its failing CI now)